src/hooks/useIpc.test.js

```diff
@@ -1,4 +1,4 @@
-import { toast } from 'react-toastify';
+/* import { toast } from 'react-toastify';
```

Comment:
I think the tests should exists?

Reply:
It will reverted once changes are approved and before merging. It was commented in order to pass lint and supply a deployment test